src/Interface/HasFilter/Instance.agda

```diff
@@ -0,0 +1,23 @@
+{-# OPTIONS --safe --no-import-sorts #-}
+open import Agda.Primitive renaming (Set to Type)
+open import Axiom.Set using (Theory)
+
+module Interface.HasFilter.Instance (th : Theory) where
+
+open import Data.Product using (_×_; _,_)
+open import Data.List using (List) renaming (filter to filterˡ)
+open import Relation.Unary using (Decidable)
+open import Axiom.Set.Map th
+open import Interface.HasFilter th
+open Theory th renaming (filter to filterˢ)
+private variable A B : Type
+
+instance
```

Comment:
There should also be an instance for `List` here, which fixes the problem that the `filter` function for lists needs to be used qualified.

Reply:
Yeah, what's weird is that I have an instance for List... but it doesn't seem to be working.  I'll keep at it...